json/Side_Text.txt

```diff
@@ -177596,191 +177596,191 @@
 		"fileID": 170
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "新人さん……<%br>どんな方がいらっしゃるのか楽しみですね！",
-		"tr_text": "",
+		"tr_text": "New recruits...<%br>I look forward to meeting them!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "？？？",
 		"tr_name": "???",
 		"jp_text": "あの！！！<%br>よ、よろしいでありますか！",
-		"tr_text": "",
+		"tr_text": "Um!!!<%br>Please it's very nice to meet you thank you!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "おわぁっ！<%br>ビックリしたー！",
-		"tr_text": "",
+		"tr_text": "Waah!<%br>Where the heck did you come from?!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "？？？",
 		"tr_name": "???",
 		"jp_text": "し、失礼しました！<%br>あの、みなさんが先輩の言っていた……",
-		"tr_text": "",
+		"tr_text": "I-I'm really so very sorry!<%br>Um, you were all talking, so...",
 		"fileID": 170
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "もしかして、あなたが<%br>「ヤスミノコフ８０００Ｃ」さん……です？",
-		"tr_text": "",
+		"tr_text": "You must be...<%br>Are you Yasminkov 8000C?",
 		"fileID": 170
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "ヤスミノコフ８０００Ｃ",
 		"tr_name": "Yasminkov 8000C",
 		"jp_text": "はい！<%br>自分がヤスミノコフ８０００Ｃであるます！",
-		"tr_text": "",
+		"tr_text": "Yes sir! Yasminkov 8000C reporting<%br>sir- I mean ma'am- I mean!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "あるますって……<%br>いきなり噛んでるけど、大丈夫か……？",
-		"tr_text": "",
+		"tr_text": "You're trippin' over yer words a lot...<%br>Is somethin' wrong...?",
 		"fileID": 170
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "ヤスミノコフ８０００Ｃ",
 		"tr_name": "Yasminkov 8000C",
 		"jp_text": "うう……すみません。<%br>早速失敗しちゃいました……",
-		"tr_text": "",
+		"tr_text": "Oh... I'm sorry.<%br>I've already made a mess of things...",
 		"fileID": 170
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "大丈夫ですよ。わたしはジェネです。<%br>そして、こちらがリーダーと、モアです！",
-		"tr_text": "",
+		"tr_text": "It's alright, really. My name is Gene.<%br>This is More, and this is Leader!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "ヤスミノコフ８０００Ｃ",
 		"tr_name": "Yasminkov 8000C",
 		"jp_text": "こ、この通り、未熟な半人前であります！<%br>ご指導、よろしくお願いするでありますっ！",
-		"tr_text": "",
+		"tr_text": "I'm so grateful that you agreed to work with someone<%br>as low as me! I hope to learn a lot from you!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "はい、こちらこそよろしくお願いします。<%br>仲良くしてくださいね。",
-		"tr_text": "",
+		"tr_text": "It's very nice to meet you, too.<%br>I'm sure we'll learn a lot from each other.",
 		"fileID": 170
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "８０００Ｃさんの育成を兼ねた複数の<%br>任務には、それぞれサポートが入ります。",
-		"tr_text": "",
+		"tr_text": "Of course, when you're training 8000C,<%br>you'll have a support team to back you up.",
```

Comment:
This line says `While you're training 8000C, a support team will be there to assist you in your mission.`